From Olivier.Bornet@puck.ch  Thu Aug 20 14:32:16 2009
From: Olivier Bornet <Olivier.Bornet@puck.ch>
Date: Tue, 18 Aug 2009 21:05:53 +0200
Subject: USB: iuu_phoenix: Don't reset the device at close
To: Greg KH <greg@kroah.com>
Cc: Alan Cox <alan@lxorguk.ukuu.org.uk>, linux-usb@vger.kernel.org, Olivier Bornet <Olivier.Bornet@puck.ch>
Message-ID: <1250622357-29765-2-git-send-email-Olivier.Bornet@puck.ch>


Resetting the device cause the device to have a new name in the /dev.

Signed-off-by: Olivier Bornet <Olivier.Bornet@puck.ch>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/iuu_phoenix.c |    4 ----
 1 file changed, 4 deletions(-)

--- a/drivers/usb/serial/iuu_phoenix.c
+++ b/drivers/usb/serial/iuu_phoenix.c
@@ -1009,11 +1009,7 @@ static void iuu_close(struct usb_serial_
 		usb_kill_urb(port->write_urb);
 		usb_kill_urb(port->read_urb);
 		usb_kill_urb(port->interrupt_in_urb);
-		msleep(1000);
-		/* wait one second to free all buffers */
 		iuu_led(port, 0, 0, 0xF000, 0xFF);
-		msleep(1000);
-		usb_reset_device(port->serial->dev);
 	}
 }
 
